dev-workflow-ui/src/ch/ivyteam/workflowui/util/ViewerUtil.java

```diff
@@ -35,14 +43,34 @@ public static boolean isViewerAllowed(ICase caze) {
    */
   public static WebLink getViewerLink(IWebStartable startable) {
     if (startable instanceof IProcessWebStartable processStartable) {
-      return ProcessViewer.of(processStartable).url().toWebLink();
+      return ProcessViewer.of(processStartable).url().theme(readThemeCookie()).toWebLink();
     }
     if (startable instanceof ICaseMapWebStartable caseMapStartable) {
       return CaseMapViewer.of(caseMapStartable).url().toWebLink();
     }
     throw new IllegalArgumentException("The provided IWebStartable does not support the generation of a viewer link.");
   }
 
+  private static Theme readThemeCookie() {
+    var request = FacesContext.getCurrentInstance().getExternalContext().getRequest();
+    if (request instanceof HttpServletRequest httpRequest) {
+      return Arrays.stream(httpRequest.getCookies())
+              .filter(cookie -> "primefaces-theme-mode".equals(cookie.getName()))
+              .map(cookie -> Theme.of(cookie.getValue()))
+              .findFirst()
+              .orElse(Theme.LIGHT);
+    }
+    return Theme.LIGHT;
+  }
+
+  private static Theme readTheme() {
```

Comment:
This function is not being used anywhere? 🤔

Reply:
Nope this is the other way to read the theme. Just as second option for the reviewer to choose 😉

Reply:
in my opinion this function looks better

Reply:
same for me